app/_data/docs_nav_konnect.yml

```diff
@@ -151,6 +151,8 @@
           url: /dev-portal/applications/enable-app-reg
         - text: Dynamic Client Registration
           items:
+            - text: Overview
+              url: /dev-portal/applications/dynamic-client-registration/overview
```

Comment:
Can you please rename the file from `overview.md` to `index.md`? Using "overview" lengthens the URL unnecessarily.